packages/core-components/src/layout/Header/Header.tsx

```diff
@@ -59,6 +59,7 @@ const useStyles = makeStyles(
     },
     leftItemsBox: {
       maxWidth: '100%',
+      width: '80%', // space for the right items
```

Comment:
Would be best to have the right items take up the content that they need, rather than a fixed with. This'll cause trouble both if more items are added and on more narrow screens